docs/graphql-api/mutations/index.mdx

```diff
@@ -11,16 +11,51 @@ keywords:
   - api mutations
 ---
 
-# GraphQL Mutations
+# Basics of Mutations
 
-Mutations are used to modify data in a database. They are analogous to the `INSERT`, `UPDATE` and `DELETE` commands in
-SQL. Hasura provides a powerful and flexible GraphQL API to perform mutations.
+## Introduction
 
```

Comment:
The content in the index files is great but I'm concerned with the old thing that users may miss it by opening the section instead of realising the folder is a page itself and clicking on it.